src/main/java/model/data/EntityID.java

```diff
@@ -0,0 +1,58 @@
+package model.data;
+
+import model.JSONable;
+
+/**
+ * General class to unify
+ * 
+ * @author Wm. Keith van der Meulen
+ *
+ */
+public abstract class EntityID implements JSONable {
+	
+	protected String id;
```

Comment:
I deleted the ID fields in the sub-classes (let me know if I missed one), so there should be no duplication. In fact, this makes it easier since all JSON objects will now have a unified name for the ID